src/layout/appHeader.tsx

```diff
@@ -1,34 +1,159 @@
-import React, {FC} from 'react'
-import {Link} from 'react-router-dom'
-import {AppBar, Toolbar, Typography, Button} from '@material-ui/core'
+import React, {FC, Dispatch, SetStateAction} from 'react'
+import {useHistory} from 'react-router-dom'
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Button,
+  makeStyles,
+  IconButton,
+  Menu,
+  MenuItem,
+  useMediaQuery,
+  useTheme,
+  Divider,
+  ListSubheader,
+} from '@material-ui/core'
+import {AccountCircle} from '@material-ui/icons'
+import MenuIcon from '@material-ui/icons/Menu'
 import {useAuth0} from '@auth0/auth0-react'
+import {routes} from '../routes'
 
-export const AppHeader: FC = props => {
-  const {isAuthenticated, loginWithPopup, logout} = useAuth0()
+interface Props {
+  sideDrawOpen: boolean
+  setSideDrawerOpen: Dispatch<SetStateAction<boolean>>
+}
```

Comment:
I would rename 
sideDrawOpen to isSideDrawOpen
and 
setSideDrawerOpen to openSideDrawer

so it's self explanatory

Reply:
Thanks Marco. I changed the names to isSideDrawerOpen and setIsSideDrawerOpen. Much clearer now I hope.